src/styles/mixins.module.scss

```diff
@@ -59,11 +66,24 @@
   color: variables[black];
 }
 
+// font weight 500
 @mixin montserratMedium {
   font-family: 'Montserrat Medium';
-  color: #231f20;
+  color: variables.$black;
 }
 
 @mixin pagePadding {
   padding: 0 1.25rem;
 }
+
+@mixin desktopMaxWidth {
```

Comment:
You can combine media queiries together so that these are all one mixin. Like so:
```
@mixin maxWidth {
max-width: var.$desktopWidth;
  @media(max-width: $tablet) {
max-width: var.$tabletWidth;
}
  @media(max-width: $mobile) {
max-width: var.$mobileWidth;
}

}
```